vm/systemSmartContracts/auction.go

```diff
@@ -665,9 +703,11 @@ func (s *stakingAuctionSC) stake(args *vmcommon.ContractCallInput) vmcommon.Retu
 		registrationData.RewardAddress = args.CallerAddr
 	}
 
+	//TODO: verify if the next two lines are correct
```

Comment:
yes, they are correct. registrationData.Epoch is not yet used.

Reply:
ok